### PR TITLE
[Large Rooms] RemoteVideoTrack switch on/off and subscriptions

### PR DESCRIPTION
**Pull request by manjeshbhargav**
**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### Description

This PR contains changes for the following tickets:

- [VIDEO-8745](https://issues.corp.twilio.com/browse/VIDEO-8745)
- [VIDEO-8746](https://issues.corp.twilio.com/browse/VIDEO-8746)
- [VIDEO-8747](https://issues.corp.twilio.com/browse/VIDEO-8747)
- [VIDEO-8748](https://issues.corp.twilio.com/browse/VIDEO-8748)

### Before review
* [x] Updated CHANGELOG.md if necessary
* [x] Added unit tests if necessary
* [x] Updated affected documentation
* [x] Verified locally with `npm test`
* [ ] Manually sanity tested running locally
* [ ] Included screenshot as PR comment (if needed)
* [x] Ready for review

### CODE REVIEW

**Review by mmartikainen — APPROVED** [review]
No specific feedback from me.

File: lib/media/track/videotrack.js [hunk 1]
```diff
@@ -83,21 +86,20 @@ class VideoTrack extends MediaTrack {
   _checkIfCanCaptureFrames(isPublishing = false) {
     let canCaptureFrames = true;
     let message = '';
-    const { enabled, readyState } = this.mediaStreamTrack;
 
-    if (!enabled) {
+    if (!this.mediaStreamTrack) {
```
Comment (charliesantos) [inline lib/media/track/videotrack.js:90]: In what scenarios `mediaStreamTrack` would be null and `_checkIfCanCaptureFrames` is called?

**Review by charliesantos — APPROVED** [review]
Looks good. Just left one question.

File: lib/media/track/videotrack.js (hunk 1 above)
Comment (manjeshbhargav) [reply lib/media/track/videotrack.js:90]: In Large Rooms, there won't be a MediaStreamTrack associated with a RemoteTrack that is switched off. In that case, `_checkIfCanCaptureFrames()` should return false.

**Pull request merged**